drivers/bridge/bridge_store.go

```diff
@@ -184,7 +184,7 @@ func (ncfg *networkConfiguration) Exists() bool {
 }
 
 func (ncfg *networkConfiguration) Skip() bool {
-	return ncfg.DefaultBridge
+	return false
```

Comment:
This would probably cause issues on daemon reload with different bridge configs (-b, -bip, -fixed-cidr...), or if docker0 bridge params were manually modified.